client/state/selectors/has-available-site-feature.js

```diff
@@ -6,7 +6,7 @@ import getSiteFeatures from 'calypso/state/selectors/get-site-features';
  * @param  {object}  state      Global state tree
  * @param  {number}  siteId     The ID of the site we're querying
  * @param  {string}  featureId  The dotcom feature to check.
- * @returns {false|string[]}    Plasns array if the feature is available. Otherwise, False.
+ * @returns {false|Object.<number, string>}    Plasns array if the feature is available. Otherwise, False.
```

Comment:
minor: `Plasns` typo

Reply:
In fact, it's a plans object, right?

```suggestion
 * @returns {false|Object.<number, string>}    Plans object if the feature is available. Otherwise, False.
```

Reply:
True, but it was that way already 😂 

We can fix this in the next PR. I suspect that we need to make this into a `string[]` again.